### PR TITLE
manage realm roles with terraform on dev env

### PR DESCRIPTION
**Pull request by filipflorek**
### Changes being made

Add realm roles to dev env. Renamed main.tf to clients.tf to improve project structure.

### Context

Manage realm roles with Terraform. Default realm roles (like offline_access, uma_authorization) are not included. Roles were imported beforehand, so `terraform plan` should not show any changes.
 
### Quality Check

- [x] Terraform plan contains only my changes, or other developers are aware that their manual changes can be overridden. [^2]

[^2]: Keep in mind that sometimes Keycloak automatically adds properties to newly created resources. `terraform plan` will show them as changes made outside of Terraform. As long as those attributes are empty and do not interfere with existing configuration, they can be ignored. Here is example of one:
Terraform

### CODE REVIEW

File: keycloak-dev/realms/moh_applications/realm-roles/manage-users/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  realm_id    = "moh_applications"
+  name        = "Manage Users"
+  description = "Provides the roles required to manage users using the USER-MANAGEMENT application including roles for all applications.  In DEV this role is provided to the Developer and Midtier teams."
+  composite_roles = [
+    var.USER-MANAGEMENT-SERVICE.ROLES["create-user"].id,
+    var.USER-MANAGEMENT-SERVICE.ROLES["manage-all-groups"].id,
+    var.USER-MANAGEMENT-SERVICE.ROLES["manage-org"].id,
+    var.USER-MANAGEMENT-SERVICE.ROLES["manage-user-details"].id,
+    var.USER-MANAGEMENT-SERVICE.ROLES["manage-user-roles"].id,
+    var.USER-MANAGEMENT.ROLES["user-management-admin"].id,
```
Comment (ahoplock) [inline keycloak-dev/realms/moh_applications/realm-roles/manage-users/main.tf:11]: I think this line could go below the rest of the var.USER-MANAGEMENT-SERVICE.ROLES.

File: keycloak-dev/realms/moh_applications/realm-roles/cgi-application-support/main.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+resource "keycloak_role" "REALM_ROLE" {
+  realm_id    = "moh_applications"
+  name        = "CGI Application Support"
+  description = "Provides the top level of access to all CGI managed applications. In DEV this role is provided to the Developer, Midtier, and QA teams."
+  composite_roles = [
```
Comment (ahoplock) [inline keycloak-dev/realms/moh_applications/realm-roles/cgi-application-support/main.tf:5]: Sorry, I think these too could also be sorted for readability.

**Review by ahoplock — CHANGES_REQUESTED** [review]
See comments (sorry).

File: keycloak-dev/realms/moh_applications/realm-roles/manage-users/main.tf (hunk 1 above)
Comment (filipflorek) [reply keycloak-dev/realms/moh_applications/realm-roles/manage-users/main.tf:11]: I'll put it on top. Sorry, updated the script to sort by role name and not the whole entry.

**Review by ahoplock — APPROVED** [review]
Reviewed and approved.

**Pull request merged**